### ISSUE TITLE
Delete DNS record fails to update PTR Zone

### ISSUE THREAD

**Issue by cyberlink1** [webmin/webmin#821]
When you go to delete a record from the zone, it does not delete the corresponding PTR record even when the checkbox is selected. 

Checking the log file I found the following error. (changing domain and IP for security reasons)

Argument "11167/Name.Domain.com./A/10.210.31.172" isn't numeric in sort at /usr/libexec/webmin/bind8/delete_recs.cgi line 43.

**Comment by jcameron** [comment]
Are your reverse zones partially delegated, or do you own an entire class C ?

**Comment by cyberlink1** [comment]
Actually I have a Class B of routable addresses and use the 10.0.0.0/8 for internal addresses. All PTR zones are configured at the class C level for manageability.

**Comment by jcameron** [comment]
So the PTR addresses are in the 10.0.0.0/8 block?

**Comment by cyberlink1** [comment]
Yes the PTR's are in a 10.0.0.0/8 block but the zones are configured at a /24 within that block.

**Comment by jcameron** [comment]
If you open that zone in Webmin's BIND module, can you post a screenshot of how the records appear?

**Comment by iliaross** [comment]
❗️ If the issue isn't resolved, feel free to reopen it with updated details.
